myrepos/mrconfig

```diff
@@ -49,6 +49,12 @@ checkout = git clone 'git@github.com:canonical/kafka-k8s-operator' 'kafka-k8s-op
 [kafka-operator]
 checkout = git clone 'git@github.com:canonical/kafka-operator' 'kafka-operator'
 
+[charmed-kafka-snap]
+checkout = git clone 'git@github.com/canonical/charmed-kafka-snap' 'charmed-kafka-snap'
+
+[charmed-kafka-rock]
```

Comment:
I see that for the other product we used the suffix `-container`. Shoud we use that ?

Reply:
At the moment we are diverged here. Let's commit is AS IS and sync naming separately. Need to ACK with with Mykola.